src/lexer.rs

```diff
@@ -20,7 +28,48 @@ pub enum ParseMode {
     /// correspond to.
     /// 1. `/` can by used as a synonym for `OR`, and doesn't need to be
     /// separated by whitespace from the terms it combines
-    Lax,
+    pub const LAX: Self = Self::STRICT
+        .allow_lower_case_operators()
+        .allow_slash_as_or_operator()
+        .allow_imprecise_license_names()
+        .allow_postfix_plus_on_gpl();
+
+    const fn strict() -> Self {
+        Self {
+            allow_lower_case_operators: false,
+            allow_slash_as_or_operator: false,
+            allow_imprecise_license_names: false,
+            allow_postfix_plus_on_gpl: false,
+        }
+    }
+
+    const fn allow_lower_case_operators(self) -> Self {
+        Self {
+            allow_lower_case_operators: true,
+            ..self
+        }
+    }
+
+    const fn allow_slash_as_or_operator(self) -> Self {
+        Self {
+            allow_slash_as_or_operator: true,
+            ..self
+        }
+    }
+
+    const fn allow_imprecise_license_names(self) -> Self {
+        Self {
+            allow_imprecise_license_names: true,
+            ..self
+        }
+    }
+
+    const fn allow_postfix_plus_on_gpl(self) -> Self {
+        Self {
+            allow_postfix_plus_on_gpl: true,
+            ..self
+        }
+    }
 }
```

Comment:
I don't really think it's necessary to add these builder methods, they just over complicate the API when they can just be directly set by field.

Reply:
yeah, I guess it's simple enough without them. should be removed now :)